docs/pages/faqs.md

```diff
@@ -2,6 +2,20 @@
 
 This section of the documentation will cover common questions and encounters often asked by users and developers.
 
+## Tooling
+
+:::note "Package manager issues"
+
+Lodestar relies on [Corepack](https://nodejs.org/api/corepack.html) and associated `packageManager` value to manage its package manager version.
+
+Make sure `Corepack` is correctly enabled if you envounter some package manager related issues:
```

Comment:
```suggestion
Make sure `corepack` is correctly enabled if you encounter some package manager related issues:
```

I am guessing if you put `` around Corepack you are referencing the installed package / cli, which is lower case